### ISSUE TITLE
yarn name conflict with Hadoop YARN when creating a new app

### ISSUE THREAD

**Issue by MayukhSobo** [gobuffalo/buffalo#1217]
![image](https://user-images.githubusercontent.com/8527634/43764873-3ac7f136-9a4c-11e8-902b-7a01bd647ae7.png)

Please help. ```yarn install``` also yields the same error

**Comment by stanislas-m** [comment]
Can you provide the result of `buffalo info`?

**Comment by MayukhSobo** [comment]
```sh
buffalo info
### Buffalo Version
v0.12.4

### App Information
Pwd=/Users/mayukhsarkar/Documents/Codes/go/src
Root=/Users/mayukhsarkar/Documents/Codes/go/src
GoPath=/Users/mayukhsarkar/Documents/Codes/go
Name=src
Bin=bin/src
PackagePkg=src
ActionsPkg=src/actions
ModelsPkg=src/models
GriftsPkg=src/grifts
VCS=
WithPop=false
WithSQLite=false
WithDep=false
WithWebpack=false
WithYarn=false
WithDocker=false
WithGrifts=false

### Go Version
go version go1.10.3 darwin/amd64

### Go Env
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/mayukhsarkar/Library/Caches/go-build"
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/mayukhsarkar/Documents/Codes/go"
GORACE=""
GOROOT="/usr/local/Cellar/go/1.10.3/libexec"
GOTMPDIR=""
GOTOOLDIR="/usr/local/Cellar/go/1.10.3/libexec/pkg/tool/darwin_amd64"
GCCGO="gccgo"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/sd/8cjnjs_12ld8fzl3yhk673z40000gn/T/go-build412185568=/tmp/go-build -gno-record-gcc-switches -fno-common"

### Node Version
v10.5.0

### NPM Version
6.1.0

### Yarn Version
ERROR: --version is not COMMAND nor fully qualified CLASSNAME.
Usage: yarn [OPTIONS] SUBCOMMAND [SUBCOMMAND OPTIONS]
 or    yarn [OPTIONS] CLASSNAME [CLASSNAME OPTIONS]
  where CLASSNAME is a user-provided Java class

  OPTIONS is none or any of:

--buildpaths                       attempt to add class files from build tree
--config dir                       Hadoop config directory
--daemon (start|status|stop)       operate on a daemon
--debug                            turn on shell script debug mode
--help                             usage information
--hostnames list[,of,host,names]   hosts to use in worker mode
--hosts filename                   list of hosts to use in worker mode
--loglevel level                   set the log4j level for this command
--workers                          turn on worker mode

  SUBCOMMAND is one of:


    Admin Commands:

daemonlog            get/set the log level for each daemon
node                 prints node report(s)
rmadmin              admin tools
scmadmin             SharedCacheManager admin tools

    Client Commands:

applicationattempt   prints applicationattempt(s) report
app|application      prints application(s) report/kill application/manage
                     long running application
classpath            prints the class path needed to get the hadoop jar and
                     the required libraries
cluster              prints cluster information
container            prints container(s) report
envvars              display computed Hadoop environment variables
jar <jar>            run a jar file
logs                 dump container logs
queue                prints queue information
schedulerconf        Updates scheduler configuration
timelinereader       run the timeline reader server
top                  view cluster information
version              print the version

    Daemon Commands:

nodemanager          run a nodemanager on each worker
proxyserver          run the web app proxy server
registrydns          run the registry DNS server
resourcemanager      run the ResourceManager
router               run the Router daemon
sharedcachemanager   run the SharedCacheManager daemon
timelineserver       run the timeline server

SUBCOMMAND may print help when invoked w/o parameters or with -h.
Usage:
  buffalo info [flags]

Flags:
  -h, --help   help for info

ERRO[0001] Error: exit status 1
```

**Comment by stanislas-m** [comment]
Interesting, `yarn` is the name of two different tools, and cause conflicts: https://github.com/yarnpkg/yarn/issues/673

**Comment by MayukhSobo** [comment]
How to fix it then? For now?

**Comment by stanislas-m** [comment]
You'll need to rename Hadoop `yarn` to something else, I'm afraid.

`yarn` javascript seems to define an alias: can you give me the output of `yarnpkg`?

**Comment by MayukhSobo** [comment]
Command not found...I can not do that I can not rename that. I use Hadoop's YARN a lot..I am rather new to web development..This was supposed to be easy.

**Comment by stanislas-m** [comment]
If you can use Docker, you can use this version in the meanwhile: https://hub.docker.com/r/gobuffalo/buffalo/

**Comment by stanislas-m** [comment]
Fixed with #1218.
